packages/components/psammead-bulleted-list/src/bullet.svg

```diff
@@ -0,0 +1,3 @@
+<svg height="20" width="20" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Is there a reason why this svg is not in `psammead-assets` folder with all of the other svgs?

Reply:
Yes. Because it needs to be imported as a url into css code, not as a React element. On a second look, it does seem to belong to psammead-assets as it is in itself an asset, so a better way might be to use `React.renderToString` and `btoa` to convert it to a b64 string for use directly in the css.

I'll update this shortly

Reply:
After a discussion with @jroebu14 it's best to keep the sag in psammead-bulleted-lists due it's specificity.